HW3-datereaderMetod/Program.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+
+namespace HW3_datereaderMetod
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.Write("Введите дату: ");
+            string userInput = Console.ReadLine();
+            bool result = DateTime.TryParse(userInput, out DateTime date);
+            if (result)
```

Comment:
This implementation can be done without additional "else". See example of class work